.travis.yml

```diff
@@ -11,12 +11,17 @@
 # travis-ci.org CI unit tests integration configuration file.
 
 language: python
-python:
-  - "2.7"
+
+matrix:
+  include:
+  - python: "2.7"
+    env: SHOTGUN_COMPILE_ONLY=0
+  - python: "3.3"
```

Comment:
the vfxplatform is planning to move to 3.6 or above - any chance we could bump it up all the way to 3.6?

Reply:
Are they? I looked on the website and didn't find anything. I'll bump it to 3.6 then.

Reply:
check out this thread: https://groups.google.com/forum/#!topic/vfx-platform-discuss/Nfsl8Uf-dpU